pyproject.toml

```diff
@@ -1,3 +1,50 @@
+[project]
+name = "PROJECT"
+authors = [
+    { name = "Jason R. Coombs", email = "jaraco@jaraco.com" },
+]
+description = "PROJECT_DESCRIPTION"
+readme = "README.rst"
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+]
+requires-python = ">=3.8"
+dynamic = ["version"]
+dependencies = [
+]
+
+[project.optional-dependencies]
+testing = [
+    # upstream
+    "pytest >= 6, != 8.1.1",
+    "pytest-checkdocs >= 2.4",
+    "pytest-cov",
+    "pytest-mypy",
+    "pytest-enabler >= 2.2",
+    "pytest-ruff >= 0.2.1",
+
+    # local
+]
+docs = [
+    # upstream
+    "sphinx >= 3.5",
+    "jaraco.packaging >= 9.3",
+    "rst.linker >= 1.9",
+    "furo",
+    "sphinx-lint",
+
+    # local
+]
+
+[project.urls]
+Homepage = "https://github.com/PROJECT_PATH"
+
+[project.scripts]
+
 [build-system]
```

Comment:
I'd be slightly tempted to put `[build-system]` at the top, since that seems to be the most essential/foundational purpose of the pyproject.toml. What do you think? It also makes it easier to reconcile differences when appending output from something like ini2toml to the pyproject.toml file.

Reply:
I fully agree with the point. I saw `[build-system]` in other projects, like [pydantic](https://github.com/pydantic/pydantic/blob/ae71183f5853336f7b3a17899db20ede3a5de5f8/pyproject.toml#L1-L4), [Ruff](https://github.com/astral-sh/ruff/blob/1801798e85e800be7462982941a1d93be176fcd0/pyproject.toml#L1-L5) or [pandas](https://github.com/pandas-dev/pandas/blob/888b6bc1bb3fdf203dda64e50336db4eb9b1e778/pyproject.toml#L1-L17) and all of them use different build systems.

Reply:
When it comes to `tool.*` sections, I do however believe their place is below the `project` section. @jaraco are you opposed to that approach?

Reply:
Yes, I think I agree, though it will make the transition more difficult, because the conversion of setup.cfg to pyproject.toml will need to insert the new settings between [build-system] and [tool.setuptools_scm].

Reply:
The sections are _almost_ alphabetical, just not quite :/